monai/bundle/scripts.py

```diff
@@ -116,6 +121,216 @@ def _get_fake_spatial_shape(shape: Sequence[Union[str, int]], p: int = 1, n: int
+
+    Args:
+        name: Bundle and weights name. If `None`, `url` should be provided, or the weights file is existing locally and
+            the default weights file should be named as "model.pt".
+            If not `None`, it should be a string that consisted with the bundle name, a separator `#`
+            and the weights name. For example: if the bundle name is "spleen" and the weights name is "model.pt", then
+            `name` is "spleen#model.pt".
+        is_ts_model, a flag to specify if the weights file is a TorchScript module.
+        bundle_dir: the directory the weights will be loaded from.
+            Default is `bundle` subfolder under`torch.hub get_dir()`.
+        source: the place that saved the bundle.
+            If `source` is `github`, the bundle should be within the releases.
+        repo: the repo name. If the weights need to be downloaded first and `url` is `None`, it must be provided.
+        url: the url to download the data.
+        progress: whether to display a progress bar when downloading.
+        map_location: pytorch API parameter for `torch.load` or `torch.jit.load`.
+        net_name: if not `None`, a corresponding network will be instantiated and load the achieved weights.
+            This argument only works when loading weights.
+        net_kwargs: other arguments that are used to instantiate the network class defined by `net_name`.
+
+    """
+    if bundle_dir is None:
+        get_dir, has_home = optional_import("torch.hub", name="get_dir")
+        if has_home:
+            bundle_dir = Path(get_dir()) / "bundle"
+        else:
+            raise ValueError("bundle_dir=None, but no suitable default directory computed. Upgrade Pytorch to 1.6+ ?")
+    bundle_dir = Path(bundle_dir)
+
+    weights_name: str = "model.pt"
+    if name is not None:
+        if len(name.split(ID_SEP_KEY)) == 2:
+            weights_name = name.split(ID_SEP_KEY)[1]
+        else:
+            raise ValueError(f"The format of `name` is wrong.\n{run.__doc__}")
+    model_file_path = os.path.join(bundle_dir, weights_name)
+
+    if not os.path.exists(model_file_path):
+        if name is None and url is None:
+            raise ValueError(f"To download and load model from source: {source}, `name` must be provided.")
+        if repo is None and url is None:
+            raise ValueError(f"To download and load model from source: {source}, `repo` must be provided.")
+        download(name=name, bundle_dir=bundle_dir, source=source, repo=repo, url=url, progress=progress)
+    # loading with `torch.jit.load`
+    if is_ts_model is True:
+        return torch.jit.load(model_file_path, map_location=map_location)
+    # loading with `torch.load`
+    model_dict = torch.load(model_file_path, map_location=map_location)
+
+    if net_name is not None:
+        net_config = {"_target_": net_name}
+        for k, v in net_kwargs.items():
+            net_config[k] = v
```

Comment:
I think here you can use `net_kwargs` as the config dict directly:
```py
net_kwargs["_target_"] = net_name
configer = ConfigComponent(config=net_kwargs)
```
What do you think?

Thanks.